supereeg/brain.py

```diff
@@ -450,7 +460,10 @@ def plot_locs(self, pdfpath=None):
         """
 
         locs = self.get_locs()
-        label = np.array(self.label)[self.filter_inds.flatten()]
+        if not self.filter_inds.all():
```

Comment:
I think the filtered electrodes should *always* be shown-- it's just that if all of the electrodes are filtered out, they'll all show up in the grayed out x color.

Also: if filtered out locations also appear in the model, those locations should be plotted in red.  Can you verify that happens?  (I.e. if a location is filtered out and then reconstructed, it shouldn't remain grayed out-- the color should indicate that it's a reconstructed recording)